include/dbgroup/atomic/mwcas/lock_free/mwcas_descriptor.hpp

```diff
@@ -285,7 +285,7 @@ class alignas(kCacheLineSize) MwCASDescriptor
    * @param target A target MwCAS information.
    * @param desired A desired value to be embedded.
    */
-  void Finalize(            //
+  bool Finalize(            //
```

Comment:
返り値の型は後置で宣言しましょう。 [色々便利なケース](https://cpprefjp.github.io/lang/cpp11/trailing_return_types.html) があります。また詳細は後述しますが返り値はカウンタの値にしましょう。

```suggestion
  auto Finalize(            //
      uint64_t desc_addr,   //
      MwCASTarget &target,  //
      uint64_t desired)  //
    -> uint64_t;
```